pkg/plugin/conntrack/_cprog/conntrack.c

```diff
@@ -123,9 +140,22 @@ static __always_inline bool _ct_create_new_tcp_connection(struct ct_v4_key key,
         return false;
     }
     new_value.eviction_time = now + CT_SYN_TIMEOUT;
-    new_value.flags_seen_tx_dir = flags;
+    new_value.flags_seen_tx_dir = p->flags;
     new_value.is_direction_unknown = false;
     new_value.traffic_direction = _ct_get_traffic_direction(observation_point);
+
+    #ifdef CONNTRACK_METRICS
```

Comment:
if `CONNTRACK_METRICS` is only defined _iff_ we actually want conntrack metrics, does it make sense here to have just the `ifdef` block only here and skip the `#if CONNTRACK_METRICS == 1`?